src/jdk.incubator.foreign/share/classes/jdk/internal/foreign/NativeMemorySegmentImpl.java

```diff
@@ -0,0 +1,113 @@
+/*
+ *  Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Update copyright year?
```suggestion
 *  Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
```